### DISCUSSION TITLE
When do I need to update other firmware?

### DISCUSSION THREAD

**Discussion by EnochPrime** [meatpiHQ/wican-fw#71] [Q&A]
Just received a WiCAN and it came with v1.69 so I wanted to update to the latest (v2.80). I noticed that the [OTA](https://github.com/meatpiHQ/wican-fw#1-ota) instructions only mention loading the `wican-fw_*.bin` file.

- Is it possible to load the bootloader and other firmware files through OTA?
- If I can pass it any `*.bin` should I only do the application software partition?
- Has these been significant changes to the bootloader et. al. since v1.69 that would require updating?

**Comment by meatpiHQ** [comment]
@EnochPrime Have you tried to update using the web interface? Just go to the 'about' tab and load the firmware and click update. No need to update the bootloader or other files, those files are still the same, you only need the main file.

**Reply by EnochPrime** [reply]
I have twice now and it was not successful. It allegedly uploads and then the page refreshes and still reports the old firmware. Tried rebooting after that and it still shows the old firmware.

**Reply by meatpiHQ** [reply]
@EnochPrime please try again using AP+Station mode, and update over station IP. 

If that fails, then flash using the USB interface https://github.com/meatpiHQ/wican-fw#2-usb-flash

**Reply by drcvach** [reply]
Hi, I have the same problem with updating my WiCAN module. I am trying to update from fw v1.71 via OTA to wican-fw_v280_hv300.bin using station IP.
But after reset the version remains the same. It even deleted the wifi setup, so something happened.
Could we try to flash older FW?

**Reply by meatpiHQ** [reply]
@drcvach yes you can try. 

I suggest to update using the USB interface for now. I'm working on fixing the ota bug.

**Reply by drcvach** [reply]
great, thank you. BTW, I managed to update to v2.50 via OTA.

**Reply by supercj70** [reply]
I was updating OTA from 1.60 and doing it in steps, ending on 2.80 (v3.00_obd) . I found that even though it flashes like its rebooting after 30 seconds, if I left it, maybe 5 minutes it would automatically refresh and reflect the version I had pushed. However, if I wasn't patient and rebooted or checked status it wouldn't ever complete the update. I didn't realize this until my last update but wanted to say something in case others have experience it too.

**Reply by meatpiHQ** [reply]
OTA fix in v2.90

[wican-fw_v290_hv300.zip](https://github.com/meatpiHQ/wican-fw/files/13365791/wican-fw_v290_hv300.zip)
[wican-fw-usb_v290_hv150.zip](https://github.com/meatpiHQ/wican-fw/files/13365792/wican-fw-usb_v290_hv150.zip)

**Reply by ggtimtom** [reply]
I had also the issue that after an update the old version is still there. After i switched off the standby mode it worked fine.

**Reply by MarcusTseng** [reply]
I can confirm the same, also disabled the sleep mode and updated the firmware successfully
